file_formats/ie_formats/ini_anim.htm

```diff
@@ -0,0 +1,849 @@
+      The following animation sequence codes are used throughout the majority of animation types:<br />
+      - <span class="mono">A1</span> &rarr; <span class="mono">SEQ_SLASH</span><br />
+      - <span class="mono">A2</span> &rarr; <span class="mono">SEQ_BACKSLASH</span><br />
+      - <span class="mono">A3</span> &rarr; <span class="mono">SEQ_JAB / SEQ_SHOOT</span><br />
+      - <span class="mono">S1</span> &rarr; <span class="mono">SEQ_SHOOT</span><br />
+      - <span class="mono">CA</span> &rarr; <span class="mono">SEQ_CAST</span><br />
+      - <span class="mono">DE</span> &rarr; <span class="mono">SEQ_DIE</span><br />
+      - <span class="mono">GH</span> &rarr; <span class="mono">SEQ_DAMAGE</span><br />
+      - <span class="mono">GU</span> &rarr; <span class="mono">SEQ_AWAKE</span><br />
+      - <span class="mono">SC</span> &rarr; <span class="mono">SEQ_READY / SEQ_HEAD_TURN</span><br />
+      - <span class="mono">SD</span> &rarr; <span class="mono">SEQ_HEAD_TURN</span><br />
+      - <span class="mono">SL</span> &rarr; <span class="mono">SEQ_SLEEP</span><br />
+      - <span class="mono">SP</span> &rarr; <span class="mono">SEQ_CONJURE</span><br />
+      - <span class="mono">TW</span> &rarr; <span class="mono">SEQ_TWITCH</span><br />
+      - <span class="mono">WK</span> &rarr; <span class="mono">SEQ_WALK</span><br />
+      - <span class="mono">EMERGE</span> &rarr; <span class="mono">SEQ_EMERGE</span> (type 3000 only)<br />
+      - <span class="mono">HIDE</span> &rarr; <span class="mono">SEQ_HIDE</span> (type 3000 only)<br />
+      <br />
+      The <em>[general]</em> section contains attributes which are common to all available animation types. However, not all available attributes are considered by all animation types because of animation type restrictions or implementation-specific reasons. The following attributes are available:<br />
+      <br />
+      <ul>
+        <li class="anchors"><strong>animation_type = 4-digits_hex_value</strong><br />This attribute expects the creature type of the animation slot as a hexadecimal number. The INI file requires a section of the specified creature type.</li>
+        <li class="anchors"><strong>move_scale = int_or_decimal</strong><br />This attribute expects the default walking speed for the creature animation. The value can be specified as integer or decimal value. The default movement speed for humanoid creatures is 9.</li>
+        <li class="anchors"><strong>ellipse = int</strong><br />This attribute expects the selection circle radius. The default radius for humanoid creatures is 16.</li>
+        <li class="anchors"><strong>color_blood = int</strong><br />This attribute expects the color gradient index for blood splatter.</li>
+        <li class="anchors"><strong>color_chunks = int</strong><br /><span class="unknown">It is likely that this attribute expects the color gradient index for the associated chunk animation.</span></li>
```

Comment:
I can't find our note on it, but from memory I'd confirm this (that they weren't using the blood color palette).

Reply:
More here, maybe they just externalized the forced setting mentioned by kjeron: https://github.com/gemrb/gemrb/issues/128